subsys/shell/modules/Kconfig

```diff
@@ -5,17 +5,24 @@
 
 config KERNEL_SHELL
 	bool "Enable kernel shell"
+	default y
+	select INIT_STACKS
```

Comment:
wonder if those should not be `imply`. There are commands in that set (`version`, `uptime`) which use may want to use but without overhead of stack_info and monitor.